trail/query/tests.py

```diff
@@ -73,3 +73,10 @@ def testBasicWcsQuery(self):
         response = self.metadataDao.queryBySpecifiedSky(queryParam)
 
         self.assertEqual(len(response), 1)
+
+    def testBasicJoinQuery(self):
+        """Tests getMeatadataInSpecifiedSky in metadataDao is functional"""
+        paramDict = {"raLow": 0, "raHigh": 1000, "decLow": 0, "decHigh": 1000, "processor_name__icontains": "fits"}
+        response = self.metadataDao.queryJoinWcsAndMetadataParam(paramDict)
+
+        self.assertEqual(len(response), 1)
```

Comment:
I think this is outdated and you can keep the version from the `main` branch

Reply:
Not sure what you mean by the "version from the `main` branch"? I think this tests the newly added functionality where one can query Metadata table on WCS conditionals.